src/server/helpers/middleware.js

```diff
@@ -139,7 +139,10 @@ export function makeEntityLoader(modelName, additionalRels, errMessage) {
 
 	return async (req, res, next, bbid) => {
 		const {orm} = req.app.locals;
-		if (commonUtils.isValidBBID(bbid)) {
+		if (req.path === '/create') {
```

Comment:
wouldn't it be better if the path is first converted to small letters? I think /work/CREATE will fail here

Reply:
Good point. i'll modify it tomorrow.
I'm currently struggling with how to mock user authentication to test the /create and /edit routes…